app/lib/features/bug_report/controllers/bug_report_controller.dart

```diff
@@ -1,8 +1,37 @@
+import 'dart:convert';
 import 'dart:io';
 
 import 'package:effektio_flutter_sdk/effektio_flutter_sdk.dart';
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
+import 'package:http/http.dart' as http;
+import 'package:http_parser/http_parser.dart';
+import 'package:path/path.dart';
+
+const rageshakeUrl = String.fromEnvironment(
+  'RAGESHAKE_URL',
+  defaultValue: 'http://localhost/api/submit',
+);
+
+const rageshakeUsername = String.fromEnvironment(
+  'RAGESHAKE_USERNAME',
+  defaultValue: 'alice',
+);
+
+const rageshakePassword = String.fromEnvironment(
+  'RAGESHAKE_PASSWORD',
+  defaultValue: 'secret',
+);
```

Comment:
Did we try if these are actually needed? The config docs imply this is only for the other `api/listings` API endpoint.

Reply:
You were correct.
Fixed by https://github.com/acterglobal/a3/pull/488/commits/e5d210a86950453c80bfbfbea240e79f2279f62f